src/Widgets/LinkContainerWidget/LinkContainerWidgetEditingConfig.ts

```diff
@@ -2,10 +2,11 @@ import { Link, provideEditingConfig } from 'scrivito'
 import { classNameToThumbnail } from '../../utils/classNameToThumbnail'
 import { LinkWidget } from '../LinkWidget/LinkWidgetClass'
 import { LinkContainerWidget } from './LinkContainerWidgetClass'
+import Thumbnail from './thumbnail.svg'
```

Comment:
For `LinkContainerWidget` there is already an icon. But there is one missing for `LinkWidget`. Please add it.